### PR TITLE
chore: fix error message

### PR DESCRIPTION
**Pull request by jiacai2050**
## Detailed Changes
- Attach endpoint to remote error

## Test Plan
CI

### CODE REVIEW

File: remote_engine_client/src/lib.rs [hunk 1]
```diff
@@ -93,12 +94,14 @@ pub mod error {
         },
 
         #[snafu(display(
-            "Failed to query from table in server, table_idents:{:?}, code:{}, msg:{}",
+            "Failed to query from table in server, table_idents:{:?}, endpoint:{}, code:{}, msg:{}",
             table_idents,
+            endpoint.to_string(),
```
Comment (ShiKaiWi) [inline remote_engine_client/src/lib.rs:99]: Isn't `Debug` trait derived by `Endpoint`?

File: integration_tests/cases/env/cluster/ddl/partition_table.result [hunk 2]
```diff
@@ -99,9 +99,10 @@ ALTER TABLE partition_table_t ADD COLUMN (b string);
...
+Failed to execute query, err: Server(ServerError { code: 500, msg: "Failed to execute plan. Caused by: Internal error, msg:Failed to execute interpreter, err:Failed to execute insert, err:Failed to write table, err:Failed to write tables in batch, tables:[\"__partition_table_t_1\"], err:Failed to query from table in server, table_idents:[TableIdentifier { catalog: \"ceresdb\", schema: \"public\", table: \"__partition_table_t_1\" }], endpoint:xx, code:401, msg:failed to decode row group payload. Caused by: Schema mismatch with the write request, msg:expect 6 columns, but got 5. sql:INSERT INTO partition_table_t (t, id, name, value) VALUES (1651737067000, 10, \"ceresdb0\", 100);" })
```
Comment (ShiKaiWi) [inline integration_tests/cases/env/cluster/ddl/partition_table.result:105]: Why is the endpoint `xx`?

**Review by ShiKaiWi — APPROVED** [review]

File: remote_engine_client/src/lib.rs (hunk 1 above)
Comment (jiacai2050) [reply remote_engine_client/src/lib.rs:99]:
```
impl ToString for Endpoint {
    fn to_string(&self) -> String {
        format!("{}:{}", self.addr, self.port)
    }
}

```
This is more readable then debug format.

File: integration_tests/cases/env/cluster/ddl/partition_table.result (hunk 2 above)
Comment (jiacai2050) [reply integration_tests/cases/env/cluster/ddl/partition_table.result:105]: IP is not a fixed value, so I replace to `xx` with this
```
-- SQLNESS REPLACE endpoint:(.*?), endpoint:xx,
```

**Pull request merged**